code/modules/jobs/job_types/lp_jobs.dm

```diff
@@ -0,0 +1,195 @@
+	uniform = /obj/item/clothing/under/rank/medical/paramedic/lp
+	alt_uniform = /obj/item/clothing/under/rank/medical/paramedic/skirt/lp
+	dcoat = /obj/item/clothing/suit/hooded/wintercoat/medical
+	shoes = /obj/item/clothing/shoes/sneakers/white
+	head = /obj/item/clothing/head/beret/med
+	backpack_contents = list(/obj/item/melee/classic_baton/telescopic=1)
+
+	backpack = /obj/item/storage/backpack/ert/medical
+	satchel = /obj/item/storage/backpack/satchel/med
+	duffelbag = /obj/item/storage/backpack/duffelbag/med
+	courierbag = /obj/item/storage/backpack/messenger/med
+	box = /obj/item/storage/box/survival/medical
+
+
+/datum/job/lp/commissioner
+	access = list()
+	minimal_access = list()
+	outfit = /datum/outfit/job/lp/commissioner
+
+/datum/job/lp/commissioner/get_access()
+	return get_lpcomm_access()
+
+
+/datum/outfit/job/lp/commissioner
+	name = "LP Commissioner"
+	jobtype = /datum/job/lp/commissioner
+
+	ears = /obj/item/radio/headset/headset_com
+	id = /obj/item/card/id/lpcomm
+	belt = /obj/item/pda/heads/lp/commissioner
+	gloves = /obj/item/clothing/gloves/color/black
+	uniform = /obj/item/clothing/under/rank/civilian/lawyer/bluesuit
+	alt_uniform = /obj/item/clothing/under/rank/civilian/lawyer/bluesuit/skirt
+	dcoat = /obj/item/clothing/suit/jacket
+	shoes = /obj/item/clothing/shoes/jackboots
+	head = /obj/item/clothing/head/beret/command
+	backpack_contents = list(/obj/item/melee/classic_baton/telescopic=1)
+
+	backpack = /obj/item/storage/backpack/captain
+	satchel = /obj/item/storage/backpack/satchel/cap
+	duffelbag = /obj/item/storage/backpack/duffelbag/captain
+	courierbag = /obj/item/storage/backpack/messenger/com
+
+
+/datum/job/scientist/lp
+	access = list(ACCESS_AWAY_GENERAL, ACCESS_AWAY_MAINT, ACCESS_LP_FACILITIES, ACCESS_LP_AI, ACCESS_ROBOTICS, ACCESS_TOX, ACCESS_TOX_STORAGE, ACCESS_RESEARCH, ACCESS_XENOBIOLOGY, ACCESS_MECH_SCIENCE, ACCESS_MINERAL_STOREROOM, ACCESS_TECH_STORAGE)
+	minimal_access = list(ACCESS_AWAY_GENERAL, ACCESS_AWAY_MAINT, ACCESS_LP_FACILITIES, ACCESS_LP_AI, ACCESS_ROBOTICS, ACCESS_TOX, ACCESS_TOX_STORAGE, ACCESS_RESEARCH, ACCESS_XENOBIOLOGY, ACCESS_MECH_SCIENCE, ACCESS_MINERAL_STOREROOM, ACCESS_TECH_STORAGE)
+	outfit = /datum/outfit/job/scientist/lp
+
+/datum/outfit/job/scientist/lp
+	ears = /obj/item/radio/headset/binary
+	id = /obj/item/card/id/scilp
+	jobtype = /datum/job/scientist/lp
+
+/datum/job/mining/lp
+	access = list(ACCESS_AWAY_GENERAL, ACCESS_AWAY_MAINT, ACCESS_LP_FACILITIES, ACCESS_MAINT_TUNNELS, ACCESS_MAILSORTING, ACCESS_CARGO, ACCESS_QM, ACCESS_MINING, ACCESS_MECH_MINING, ACCESS_MINING_STATION, ACCESS_MINERAL_STOREROOM)
+	minimal_access = list(ACCESS_AWAY_GENERAL, ACCESS_AWAY_MAINT, ACCESS_LP_FACILITIES, ACCESS_MAINT_TUNNELS, ACCESS_MAILSORTING, ACCESS_CARGO, ACCESS_QM, ACCESS_MINING, ACCESS_MECH_MINING, ACCESS_MINING_STATION, ACCESS_MINERAL_STOREROOM)
+	outfit = /datum/outfit/job/miner/lp
+
+/datum/outfit/job/miner/lp
+	ears = /obj/item/radio/headset/binary
+	id = /obj/item/card/id/minelp
```

Comment:
what is the purpose of these id's?

Reply:
Description, icon, name.
![image](https://user-images.githubusercontent.com/55075747/178065861-90f94355-b49b-40e3-ab99-64ba79944797.png)

Reply:
The name will get set by the job.
The desc is useless.
The icon you can just use centcom id cards

Reply:
I'd still like to have the nanotrasen icon, but I can just find which one that is and change their ID to that, since they'll all have their access written properly.

Man... Fuck jobcode.

Reply:
I looked in the code and the mapper for any ID that uses the "nanotrasen" icon, they all use the centcomm officer one.
The one on the right has stripes that denote authority and a description on the card that denotes being sent from centcomm. I don't want that. The one on the right is only used in these ID's.
![image](https://user-images.githubusercontent.com/55075747/178189845-f84c8ff0-1a4e-4f1a-9c26-8e37c52edebd.png)

Reply:
Realizing that the icons won't apply properly, I just removed these ID's and am using their bases.